src/components/application_manager/rpc_plugins/sdl_rpc_plugin/test/commands/hmi/tts_get_language_response_test.cc

```diff
@@ -105,6 +105,23 @@ TEST_F(TTSGetLanguageResponseTest, Run_LanguageNotSet_SUCCESS) {
   command->Run();
 }
 
+TEST_F(TTSGetLanguageResponseTest,
+       onTimeOut_Run_ResponseForInterface_ReceivedError) {
+  MessageSharedPtr msg = CreateMessage();
+  (*msg)[strings::params][hmi_response::code] =
+      hmi_apis::Common_Result::ABORTED;
+
+  std::shared_ptr<TTSGetLanguageResponse> command(
+      CreateCommand<TTSGetLanguageResponse>(msg));
+
+  EXPECT_CALL(mock_hmi_capabilities_,
+              RemoveFromDefaultInitializedCapabilities(hmi_apis::FunctionID::TTS_GetLanguage))
+      .Times(2);
+
+  command->Run();
```

Comment:
@ydementieiev Is it a JOKE? Still no Init() call...... :bomb:

Reply:
@AByzhynar add [everywhere](https://github.com/smartdevicelink/sdl_core/pull/3269/commits/a63d278ff6aea958f03c176be3f160a7dc6b2f24)

Reply:
@ydementieiev Check it AGAIN !!! :fire: :bomb: 

```cpp
TEST_F(TTSGetLanguageResponseTest,
       onTimeOut_Run_ResponseForInterface_ReceivedError) {
  MessageSharedPtr msg = CreateMessage();
  (*msg)[strings::params][hmi_response::code] =
      hmi_apis::Common_Result::ABORTED;

  std::shared_ptr<TTSGetLanguageResponse> command(
      CreateCommand<TTSGetLanguageResponse>(msg));

  EXPECT_CALL(mock_hmi_capabilities_,
              RemoveFromDefaultInitializedCapabilities(
                  hmi_apis::FunctionID::TTS_GetLanguage))
      .Times(2);

  command->Run();
  command->onTimeOut();
}
```

Looks that your eyes are your enemies)